src/Services/AdminSubscriptionBillingWorkflow.php

```
@@ -0,0 +1,466 @@
+            if ($pricePerSeat > 0) {
+                $reductionText = ' Your next invoice will be $' . number_format(($pricePerSeat * $seatsToRemove) / 100, 2) . ' lower.';
+            }
+        } else {
+            $stripeSubscriptionId = (string) ($sub['stripe_subscription_id'] ?? '');
+            if ($stripeSubscriptionId === '') {
+                return ['ok' => false, 'error' => 'No Stripe subscription is linked to this organisation.'];
+            }
+
+            try {
+                $updated = $stripeFactory()->updateSubscriptionSeatQuantity($stripeSubscriptionId, $newLimit);
+                Subscription::updateSeatLimit($this->db, $orgId, $newLimit);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\Subscription' in method 'removeSeats'. (undefined)

(StaticAccess)